lib/index.js

```diff
@@ -0,0 +1,122 @@
+module.exports = {
+    env: {
+        browser: true,
+        es2021: true,
+        node: true,
+    },
+    extends: ["airbnb-base", "airbnb-typescript/base", "prettier"],
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true,
+        },
+        ecmaVersion: 12,
+        sourceType: "module",
+        project: ["./tsconfig.json"],
+    },
+    plugins: ["@typescript-eslint", "prettier", "import", "sort-export-all"],
+    rules: {
```

Comment:
For main prezly repo, I think we can actually use this config and then override rules specific to that repo.
It looks like majority of rules is the same, the main difference I see are in parser options.